ui/src/shared/components/LayoutCellNote.tsx

```diff
@@ -52,6 +57,18 @@ class LayoutCellNote extends Component<Props> {
     )
   }
 
+  private get noteIconStyle(): CSSProperties {
+    const {cellBackgroundColor, cellTextColor} = this.props
+
+    if (cellBackgroundColor === DEFAULT_CELL_BG_COLOR) {
+      return {}
```

Comment:
could you return null in this case? 
if so then can just return  `{color: cellTextColor}` in case of `cellBackgroundColor !== DEFAULT_CELL_BG_COLOR`